schema/bom-1.6.schema.json

```diff
@@ -661,34 +661,35 @@
             {
               "type": "array",
               "title": "Creation Tools (legacy)",
-              "description": "[Deprecated] The tool(s) used in the creation of the BOM.",
+              "description": "[Deprecated] The tool(s) used in the creation of the CycloneDX document.",
               "items": {"$ref": "#/definitions/tool"}
             }
           ]
         },
         "authors" :{
           "type": "array",
           "title": "Authors",
-          "description": "The person(s) who created the BOM. Authors are common in BOMs created through manual processes. BOMs created through automated means may not have authors.",
+          "description": "The person(s) who created the CycloneDX document.\nAuthors are common in documents created through manual processes. Documents created through automated means may not have authors.\nThis may be different from the author(s) of the component that the the CycloneDX document describes.",
           "items": {"$ref": "#/definitions/organizationalContact"}
         },
         "component": {
           "title": "Component",
-          "description": "The component that the BOM describes.",
+          "description": "The component that the the CycloneDX document describes.",
           "$ref": "#/definitions/component"
         },
         "manufacture": {
           "title": "Manufacture",
-          "description": "The organization that manufactured the component that the BOM describes.",
+          "description": "The organization that manufactured the CycloneDX document.\nThis may be different from the manufacturer of the component that the the CycloneDX document describes.",
```

Comment:
While we're at it: Does it have to be an organization?
What if it is an individual?

Reply:
yes it is currently designed to be an organization.
Only documenting the current status-quo.